common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -651,10 +651,15 @@ can_do_party = {
 	has_invader_title_trigger = no
 }
 can_be_sacrificed_trigger = {
-	is_adult = yes
+	is_alive = yes
+	is_dying = no
 
+	is_untouchable_trigger = no
 	is_dark_being_trigger = no
```

Comment:
MORE SACRIFICES!
```suggestion
	NOT = {
		trait = being_demon
		trait = creature_lich
	}
	is_nraqi_trigger = no
```

Reply:
Well, I can change it, but are you sure about that?
I also need to change localization, since undead kinda don't have "beating heart"

Reply:
Allowed void beings, except nraqi, but still need your opinion about undead.

Reply:
It's fine. Our undead except the liches can be imagined having "beating hearts" since they have blood.
https://wow.gamepedia.com/Undead#Biology

Reply:
Ok, added